edwin/client/static/js/components/TimelineApp.js

```diff
@@ -78,24 +69,31 @@ export default class TimelineApp extends React.Component {
  */
 class BugRow extends React.Component {
   render() {
+    let bug = this.props.bug;
+    let prs = this.props.prs;
+    let bugUrl = `https://bugzilla.mozilla.org/show_bug.cgi?id=${bug.get('id')}`;
+
     return (
       <tr>
         <td className="BugTable__data--number">
-          <a href={`https://bugzilla.mozilla.org/show_bug.cgi?id=${this.props.id}`}>
-            {this.props.id}
+          <a href={bugUrl}>
+            {bug.get('id')}
           </a>
         </td>
         <td className="BugTable__data">
-          {this.props.summary}
+          {bug.get('summary')}
         </td>
         <td className="BugTable__data">
-          <AssignedTo {...this.props.assigned_to_detail}/>
+          <AssignedTo user={bug.get('assigned_to_detail')}/>
         </td>
         <td className="BugTable__data">
-          {this.props.status}
+          {bug.get('status')}
+        </td>
+        <td className="BugTable__data--number">
+          {bug.getIn(['whiteboard_parsed', 'p'])}
         </td>
         <td className="BugTable__data--number">
-          {this.props.whiteboard_parsed.p}
+          {prs.map((pr) => <a href={pr.get('html_url')}>#{pr.get('number')}</a>)}
```

Comment:
I was thinking about how we do PRs in ernest and instead of just showing the currently open prs, maybe we should instead show all the prs for this bug and some indication of their state (open/closed).

I don't think we need to change that in this pr, but it's something I've been thinking about.